mantidproject/mantid
# Fix Python unit tests under Python 3

Fixes all Python unit tests so that they are compatible with Python 3.

**To test:**

The build servers will check the Python 2 unit tests so they should still work.To check for Python 3,
build the framework against Python 3 and run `ctest -R ^PythonInterface` -  all of these tests should now pass.

No issue number.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
